images/ami/ubuntu-2204.yaml

```diff
@@ -0,0 +1,17 @@
+---
+build_name: "ubuntu-22"
+packer_builder_type: "amazon"
+python_path: ""
+# The latest cloud-init version is unable to run #boothook created by CAPA https://github.com/kubernetes-sigs/cluster-api-provider-aws/issues/5115
+# This is a workaround to downgrade to older cloud-init version.
+# https://github.com/kubernetes-sigs/image-builder/pull/1590/files
+pinned_debs: ["cloud-init=23.1.2-0ubuntu0~22.04.1"]
```

Comment:
I guess we still need this cloud-init version?

Reply:
yeah. this is still not fixed in CAPA

Reply:
I am using OS specific tag though. `ubuntu0~22.04.1`